Files/UserControls/OngoingTasksFlyout.xaml

```diff
@@ -127,6 +127,21 @@
                 FontSize="14"
                 FontWeight="SemiBold"
                 Text="{helpers:ResourceString Name=OngoingTasks}" />
+
+            <Button
+                x:Name="DismissAllBannersButton"
+                x:Uid="OngoingTasksDismissAllBannersButton"                               
+                Height="32"                                
+                AutomationProperties.Name="Dismiss All"
+                Background="Transparent"
+                BorderBrush="Transparent"
+                Click="DismissAllBannersButton_Click"
+                ToolTipService.ToolTip="Dismiss All" 
+                HorizontalAlignment="Right">
+                <Button.Content>
+                    <TextBlock Text="Dismiss All" />
+                </Button.Content>
+            </Button>
```

Comment:
```suggestion
            <Button
                x:Name="DismissAllBannersButton"                       
                Height="32"                                
                Text="{helpers:ResourceString Name=ClearAll}"
                Background="Transparent"
                BorderBrush="Transparent"
                Click="DismissAllBannersButton_Click"
                ToolTipService.ToolTip="Dismiss All" 
                HorizontalAlignment="Right"/>

Reply:
Can you add a string called "ClearAll" to the en-us resw file?

Reply:
> Can you add a string called "ClearAll" to the en-us resw file?

Sure, done.